### ISSUE TITLE
Add BNPL Payment Method Messaging to the dUPE BNPL radio button elements

### ISSUE THREAD

**Issue by pierorocca** [Automattic/woocommerce-payments#8143]
### Description
BNPL is an important payment method that allows shoppers to spread payments over a longer time period. As a new category of online payment, consumer adoption and awareness continues to be in a growth phase. Thus, how this category of payment method is presented to shoppers is important to adoption and perception.

In the initial implementation of BNPL, [payment method messaging](https://stripe.com/docs/payments/payment-method-messaging) was added to the product page. This messaging e.g. As low as $14/month or 0% APR is critical to driving shopper awareness, usage, and conversion.

The problem is that not all shoppers will click into the product page to see that messaging. Even if they do see the messaging, it may be forgotten by the time the shopper completes filling up their basket/cart. 

WooPayments lacks that messaging at other important touch points such as the cart and checkout pages.

This feature adds the Stripe payment method messaging element to the checkout dUPE BNPL radio checkout buttons.

### Acceptance criteria
1. The dUPE element displays the payment method messaging copy for each BNPL radio button per the design, and each BNPL option used the appropriate `paymentMethodTypes` corresponding to the BNPL brand
2. The messaging language matches the browser locale language
3. TBD - Display text only messaging below the checkout Total amount

### Designs
Design needed - @nikkivias I'll connect with you directly with a few options for consideration

### Testing instructions
1. Enable Affirm on your store
2. Shop and add a simple $50+ product to the cart
3. Proceed to checkout
4. Observe that the Affirm radio button in the payment component displays the payments method message
5. Repeat for Afterpay/Clearpay and Klarna
6. Repeat testing on mobile 
7. Change the browser locale to a supported non en-US language and observe that the copy language matches the new locale.
8. TBD - Repeat above for Acceptance Criteria 3.

### Additional context
Inspirational examples

![Image](https://github.com/Automattic/woocommerce-payments/assets/109928953/9b36df24-af04-4348-b913-b4b1968cc07f)

![Image](https://github.com/Automattic/woocommerce-payments/assets/109928953/2dcc6dfb-06bc-4e94-9d8b-4fed3c05b2a5)

![Image](https://github.com/Automattic/woocommerce-payments/assets/109928953/ba59e33e-311b-46ac-aa99-045433acf860)

![Image](https://github.com/Automattic/woocommerce-payments/assets/109928953/0b6a941e-f1bb-4ebf-a07e-891f12ce205b)

cc @FangedParakeet

**Comment by frosso** [comment]
@pierorocca @nikkivias would the implementation consider both shortcode and block-based checkout? Or just block-based checkout?

The block checkout doesn't allow us to use a secondary label, however I reached out here ( p1707304672319879-slack-C02TS23QJ1X ) for suggestions.

**Comment by pierorocca** [comment]
Ideally it would be both @frosso. I had inquired with Samir if Blocks was possible or not. Ok to start with Shortcode and either add Blocks later or get this on Blocks roadmap.

**Comment by FangedParakeet** [comment]
@frosso, if we split this into separate issues for the shortcode and blocks checkouts, do you think that work could be completed in parallel or would one implementation build off the other? 🤔 Happy to divide up this work in either case.

**Comment by frosso** [comment]
@FangedParakeet good question! I'm not sure if there are any commonalities between the two 🤔 but it does feel like they would be separate enough 🤷 
I think that it could be up to the person implementing to decide how to proceed - my question was meant merely as clarification to ensure that we're not missing anything along the way :)

**Comment by FangedParakeet** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Heisenberg-63586f2ec60359001310fa6a/issues/Automattic/woocommerce-payments/8143?planning-poker) with Zenhub @brettshumaker

**Comment by pierorocca** [comment]
Looks amazing all. Well done. Huge improvement to checkout, to the BNPL offering, to the shopper messaging and discovery.
